src/__experimental__/components/input-icon-toggle/input-icon-toggle.style.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import BaseTheme from 'style/themes/base';
+
+const InputIconToggleStyle = styled.label`
+  color: ${getColor};
+  background-color: ${getBackground};
+  border-left: 1px solid #668592;
+  cursor: pointer;
+  padding: 6px;
+  margin: -2px -6px; /*correction for parent element padding*/
+`;
+
+InputIconToggleStyle.defaultProps = {
+  theme: BaseTheme
+};
+
+function getColor({ theme, isHovered, disabled }) {
+  const { colors } = theme;
+
+  if (disabled) {
+    return colors.text.disabled;
+  }
+
+  if (isHovered) {
+    return colors.text.white;
```

Comment:
I don't think that this is a valid value on the theme object.

Reply:
it matches the base theme config:
https://github.com/Sage/carbon/blob/master/src/style/themes/base/base-theme.config.js

Reply:
`colors.text.white` will return `undefined`

Reply:
you're right, fixed :)